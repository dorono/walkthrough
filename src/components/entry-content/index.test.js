import React from 'react';
import {shallow} from 'enzyme';
import EntryContent from './index';

const defaultProps = {
    data: [],
};

describe('EntryContent', () => {
    it('should render without errors', () => {
        shallow(<EntryContent {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<EntryContent {...defaultProps} />)).toMatchSnapshot();
    });
});
