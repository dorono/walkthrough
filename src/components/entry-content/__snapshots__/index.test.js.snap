// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EntryContent should match snapshot 1`] = `
<EntryContent
  data={
    Array [
      Object {
        "label": "raw",
        "value": "",
      },
      Object {
        "label": "hex",
        "value": "",
      },
      Object {
        "label": "base64",
        "value": Array [],
      },
    ]
  }
/>
`;
